README.md

```diff
@@ -110,6 +110,15 @@ Phantoms will no longer spawn if the hostile mobcap is full. This is per player.
 - Required options: `true`, `false`
 - Categories: `FEATURE`, `GILLY7CE-CARPET-ADDON`
 
+### spectatorPortalFix
```

Comment:
I'm unsure about the name `spectatorPortalFix`. How about `spectatorPlayersUsePortals`?